.travis.yml

```diff
@@ -0,0 +1,32 @@
+language: python
+python:
+  - "2.7"
+sudo: required
+services:
+  - docker
+env:
+  global:
+    - PA_TEST_ONLINE_INSTALLER=true
+  matrix:
+    - OTHER_TESTS=true
+    - PRODUCT_TEST_GROUP=0
+    - PRODUCT_TEST_GROUP=1
+    - PRODUCT_TEST_GROUP=2
+    - PRODUCT_TEST_GROUP=3
+    - PRODUCT_TEST_GROUP=4
+    - PRODUCT_TEST_GROUP=5
+    - PRODUCT_TEST_GROUP=6
+    - PRODUCT_TEST_GROUP=7
+install: 
+  - pip install --upgrade pip==6.1.1
+  - pip install -r requirements.txt
+script: 
```

Comment:
I just pushed #160 so that when we have future infra issues, we don't fail to build the base image for every test case. This means that the base image needs to be built prior to running the test cases. You can do this with `make docker-images`, I took a quick look at the travis docs, it looks like that should happen in a `before-script` section so that if it fails we won't try to run the product tests.